common/src/thrift/file_system_master.thrift

```diff
@@ -23,6 +23,7 @@ struct FileInfo {
   18: string userName
   19: string groupName
   20: i32 permission
+  21: string persistenceState
```

Comment:
Why would it make the backcompat handling easier? Is it because you want to version it?